samples/map/README.md

```diff
@@ -0,0 +1,10 @@
+**Map**
+
+The Map pattern consists on the fragmentation of a problem in independent subtasks that are resolved using the same function. The individual results are merged at the end.
+
+The Map pattern has two functions:
+
+1. Generating function
+2. Task function
+
+In this folder is possible to find different examples of this pattern.
```

Comment:
This does not add information. Remove.

Instead you can say the following:

This directory offers the following examples:

* **program_name**: One line description.